tinydrive/tinydrive-api/standalone.md

```diff
@@ -83,6 +142,40 @@ tinydrive.start({
 });
 ```
 
+## `tinydrive.controls.start`
+
+Start is similar to pick and browse, the main difference is that users do not have a way of closing the dialog with this method. This can be useful when you want to launch {{site.cloudfilemanager}} from an url where there is no application to insert the files into. This might be useful when you want to launch {{site.cloudfilemanager}} from an url but not as part of a bigger application. A [Control Object](#controlobject) is returned which control the {{site.cloudfilemanager}} lifecycle, which it no longer does by itself. For a complete list of available settings to pass in to this method see [Picker settings](#pickersettings).
+
+### Example: Using `tinydrive.controls.start`
+
+```js
+var controls = tinydrive.start({
+  token_provider: '/your-local/jwt-provider'
+}, function() {
+  setTimeout(function() {
+    controls.close();
+  }, 0);
+}, function(error) {
+  console.log(error);
+
+  setTimeout(function() {
+    controls.close();
+  }, 0);
+});
+```
+
+## Control Object
+
+When using callback-based methods a control object is returned by the function. This object is used to control the {{site.cloudfilemanager}} lifecycle for the view generated by the call and care must be taken to close the view through this, or it will remain open indefinitively.
```

Comment:
```suggestion
When using callback-based methods, a control object is returned by the function. This object is used to control the {{site.cloudfilemanager}} lifecycle for the view generated by the call. The view must be closed using the control object or it will remain open indefinitively.
```